src/App.css

```diff
@@ -1,26 +1,49 @@
+@import url('https://fonts.googleapis.com/css2?family=Montserrat&family=Roboto+Slab&display=swap');
+
 * {
   margin: 0;
   padding: 0;
   box-sizing: border-box;
 }
 
+body {
+  background-color: #fafafa;
+}
+
+/*  Navbar */
 .navBar {
-  height: 4rem;
   display: flex;
-  align-items: center;
-  padding: 0 2rem;
+  flex-direction: row;
   justify-content: space-between;
+  position: relative;
+  align-items: center;
+  font-family: 'Montserrat', sans-serif;
+  padding: 1% 5%;
+  background-color: #fff;
+  width: 100%;
+}
+
+.navTitle {
+  font-weight: bold;
+  color: #0290ff;
+  width: 15rem;
+  height: 2.313rem;
+  margin: 0.375rem 2.938rem 0.125rem 0;
 }
 
 .navItems {
   display: flex;
-  gap: 3rem;
+  flex-direction: row;
 }
 
 .navLinks {
   display: flex;
-  gap: 2rem;
+  flex-direction: row;
+  justify-content: space-between;
   align-items: center;
+  letter-spacing: 1.9px;
+  width: 13.563rem;
+  list-style: none;
 }
```

Comment:
- [ ] Please 🙏🏽 align the links, use the correct font styles, and colors to be similar to the Zeplin design.